modules/border/index.css

```diff
@@ -25,18 +27,16 @@
   border-left-width: var(--border-width);
 }
 
-.border-none { border: 0 }
-
 .rounded { border-radius: var(--border-radius) }
 .circle  { border-radius: 50% }
 
+.not-rounded { border-radius: 0 }
```

Comment:
To remove any default border-radius and apply border radius in just some parts. Useful for input groups.